server/middleware/setUpWebSession.ts

```diff
@@ -29,16 +29,6 @@ export default function setUpWebSession(): Router {
     }),
   )
 
-  // Update a value in the session so that redis resets the session TTL.
-  // Only changes every minute so that it's not updated with every request.
-  // This also has the effect of sending the set-cookie header to the browser
-  // which keeps the session alive there, although this is also taken care of
-  // by using rolling: true in the configuration above.
-  router.use((req, res, next) => {
-    req.session.nowInMinutes = Math.floor(Date.now() / 60e3)
```

Comment:
you should get rid of it from the session type too in `server/@types/express/index.d.ts`

Reply:
Done